extensions/BugModal/web/bug_modal.css

```diff
@@ -192,14 +192,11 @@ a.activity-ref {
   width: 100%;
 }
 
-.module .field .value input {
+.module .field .value input[type="text"],
+.module .field .value input:not([type]) {
   width: 100%;
```

Comment:
This has caused the dependency and regression fields to look bad. They are now all the way to the edge of the module boundary. This will need to be fixed.

![image](https://user-images.githubusercontent.com/826315/58341863-41869080-7e1d-11e9-9839-b0143cac1689.png)

Reply:
Other right side text fields such as URL look fine to me.

![image](https://user-images.githubusercontent.com/826315/58342779-56fcba00-7e1f-11e9-8691-1d2eb733c99e.png)

Reply:
Looks like the problem is `.module .field .value.edit { width: 100%; }`. I'll remove it.